sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,9 @@
+.breadcrumbs li::after {
```

Comment:
I'm not sure I prefer it, just a habit from how I generally do breadcrumbs. Since the "/" has no value beyond a presentational aspect I keep it out of the markup.